airgun/views/smart_class_parameter.py

```diff
@@ -33,7 +33,7 @@ class SmartClassParameterContent(View):
         locator=".//input[contains(@name, '[puppetclass_id]')]")
     override = Checkbox(
         locator=".//input[contains(@name, '[override]') and @type!='hidden']")
-    key_type = Select(locator=".//select[contains(@name, '[key_type]')]")
+    parameter_type = Select(locator=".//select[contains(@class, 'parameter_type')]")
```

Comment:
I didn't notice it earlier: why did you change `name` to `class` here?

I see that both work, but `name` seems a better choice. Other fields on that page don't have their name among classes, so I suppose selecting by class might turn out to not be as stable.

Reply:
good catch just did not notice name property